### PR TITLE
Run JupyterHub directly, update to 0.7.2 (IDR-0.3.7)

### PR DESCRIPTION
**Pull request by manics**
This installs and runs JupyterHub directly on a centos 7 server instead of using docker-in-docker.

Also updates to JupyterHub 0.7.2

See https://trello.com/c/0SacUNN6/24-update-all-jupyterhub-components

### CODE REVIEW

**Comment by manics** [comment]
This repo already includes a molecule test. If you use Vagrant with VirtualBox you can bring up a JupyterHub server, port-forward 8000, log in with `user`/`ome` and you should have a running notebook, e.g.

    molecule test --driver vagrant --destroy never

The molecule test does not verify that a notebook is actually running (this would require figuring out several GET/POST messages).

**Comment by manics** [comment]
I've force pushed to get rid of the commits from https://github.com/IDR/ansible-role-idr-jupyter/pull/2#issuecomment-297735027 since it didn't work as expected.

File: defaults/main.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ idr_jupyter_hub_extra_docker_args: ""
 idr_jupyter_hub_image: "imagedata/jupyterhub-githubauth:latest"
 # Docker Swarm requires a modified version of DockerSpawner:
 #idr_jupyter_hub_image: "manics/jupyterhub-githubauth:latest"
-idr_jupyter_notebook_image: "imagedata/jupyter-docker:latest"
+idr_jupyter_notebook_image: "manics/jupyter-docker:merge"
```
Comment (joshmoore) [inline defaults/main.yml]: I assume we're rolling this back before merging?

Comment (manics) [reply defaults/main.yml]: Yes, but https://github.com/IDR/jupyter-docker/pull/7 is required for testing interactively with vagrant (testing with docker uses a non-functional dummy image to save disk space on travis).

Comment (joshmoore) [reply defaults/main.yml]: Shall we get 7 in and tag?

Comment (manics) [reply defaults/main.yml]: Reverted

**Comment by manics** [comment]
Forgot to mention this is a breaking change, so `2.0.0`

**Comment by manics** [comment]
I'll have a go at making this run in a virtualenv in case it needs to be run elsewhere.

**Comment by manics** [comment]
Updated.

**Comment by joshmoore** [comment]
Tagging as 2.0.0

**Pull request merged**